packages/rlp/src/index.ts

```diff
@@ -1,16 +1,12 @@
-export type Input = string | number | bigint | Uint8Array | List | null | undefined
+export type Input = string | number | bigint | Uint8Array | Array<Input> | null | undefined
 
-// Use interface extension instead of type alias to
-// make circular declaration possible.
-export interface List extends Array<Input> {}
+export type NestedUint8Array = Array<Uint8Array | NestedUint8Array>
 
 export interface Decoded {
   data: Uint8Array | NestedUint8Array
   remainder: Uint8Array
 }
 
-export interface NestedUint8Array extends Array<Uint8Array | NestedUint8Array> {}
-
```

Comment:
I didn't see any type issues changing to circular declaration here, so perhaps TypeScript added support for this from when the comment was originally written.